docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,144 @@
+<!--  -->
+
+## XML Tag Auto Close
+
+XML tag `auto-close` is supported:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag Slash](../images/Features/AutoCloseTagSlashXML.gif)
+
+<!-- Using a `>` at the end of an opening tag generates the corresponding closing tag.
+
+![Auto Close Tag End](../images/Features/AutoCloseTagEndXML.gif) -->
+
+<!--  -->
+
+## Linked Editing
+
+Linked editing is supported, allowing for simultaneous changes an opening and closing tag pair.
+
+![Linked Editing](../images/Features/LinkedEditingXML.gif)
+
+<!--  -->
+
+## Jump to Start/End Tag
+
+For quick navigation, you can use `Ctrl + Click` on a closing tag to jump to the corresponding opening tag.
+
+![Jump To Opening Tag](../images/Features/JumpToOpeningTagXML.gif)
+
+The same applies to jumping from an opening tag to the respective closing tag.
+
+![Jump To Closing Tag](../images/Features/JumpToClosingTagXML.gif)
+
+<!--  -->
+
+## Rename Tag
+
+To rename a single tag and its corresponding opening/closing, highlight the tag and press `F2` or use the dropdown to select `Rename Symbol`.
+
+![Rename Tag](../images/Features/RenameTagXML.gif)
+
+<!--  -->
+
+## Completion Support
+
+### Basic completion
+
+If an XML file is not associated to a [bound](https://en.wikipedia.org/wiki/XML_data_binding) grammar file, you can utilize XML completion which uses existing XML tag elements.
+
+### Completion based on grammar
+
+#### Completion based on XSD
+
+When a XML file is associated with a XSD file, there is support for completion base on the tags, attribute names and values defined in the XML Schema/XSD file.
+
+![Completion Based On XSD](../images/Features/CompletionBasedOnXSD.gif)
+
+#### Completion based on DTD
+
+When XML file is associated with a DTD file, completion based on tags, attributes name and value defined in the DTD file are supported.
+
+![Completion Based On DTD](../images/Features/CompletionBasedOnDTD.gif)
+
+<!--  -->
+
+## Jump to Grammar Definition
+
+If an XML file is associated with an XSD file, [vscode-xml](https://github.com/redhat-developer/vscode-xml) supports jumping from the XML tag to the schema defintion.
+
+![Jump to XSD Def From XML](../images/Features/JumpToXSDDefFromXML.gif)
+
+<!--  -->
+
+## Hover Support
+
+### Hover based on grammar
+
+#### Hover based on XSD
+
+When XML file is associated with a XSD file, hover documentation based on tags, attributes name and value defined in the XML Schema/XSD file are supported.
+
+![Hover Based On XSD](../images/Features/HoverBasedOnXSD.gif)
+
+#### Hover based on DTD
+
+When XML file is associated with a DTD file, hover documentation based on tags, attributes name and value defined in the DTD  file are supported.
+
+![Hover Based On DTD](../images/Features/HoverBasedOnDTD.gif)
+
+<!-- -->
+
+## Validation Support
+
+### Syntax validation
+
+Syntax rules and validation for XML is supported and corrected.
```

Comment:
I believe the wording here could be improved.

Maybe something along the lines of "LemMinX will show syntax errors in your XML documentations, and provides quick fixes to resolve these errors"?